### PR TITLE
adding reflections blog

### PR DESCRIPTION
**Pull request by CoffeeGeek101**
#### Type of contribution
- [x] Code
- [ ] Documentation

#### Reference: Issue or Pull Request (PR)
- [x] Towards #64   [this will keep reference issue or PR open]  <=== _When in doubt, pick this one._
- [ ] Fixes #xxx. See also #xxx   [this will close the referenced issue or PR]
- [ ] Resolves #xxx, #xxx [for multiple issues, this will close them]

## Description: What does this implement/fix? Explain your changes.

- Changed the filename of `index-8.html` to `sprint_reports.html`

#### Reminders

- [x] Did you review the [CONTRIBUTING GUIDE](https://github.com/data-umbrella/data-umbrella-website/blob/main/CONTRIBUTING.md)?
- [ ] Did you reference an issue or another pull request?

#### Any other comments?
Made some initial changes to open the PR, Please let me know if any initial setup changes are required or not. I'll make the next changes in the coming days.

### CODE REVIEW

**Comment by CoffeeGeek101** [comment]
Okay, @SangamSwadiK. Thank you for instructions, working on it now.

**Comment by CoffeeGeek101** [comment]
@SangamSwadiK can you mention the filters for article to be added into Sprint Reports.

**Comment by reshamas** [comment]
> @SangamSwadiK can you mention the filters for article to be added into Sprint Reports.
> Also, the hero picture for the mentioned article has size of 365 x 405 and a approx 500px margin on it's both side, which affecting the article card on the webpage.
> Can you provide a larger size picture?

@SangamSwadiK 
We'll need to write a comment at the top on what the size of the images we are using, so we're consistent.

**Comment by CoffeeGeek101** [comment]
Okay, I'll wait for PR https://github.com/data-umbrella/data-umbrella-website/pull/68 to be merged.

File: snowlake-v1.2/open-source/sprint_reports.html [hunk 1]
```diff
@@ -71,6 +71,15 @@
         - scikit-learn
         - Data-Umbrella
         - Sprint-report
+
+    - title: "Reflections on the Data Umbrella PyMC February 2022 Sprint"
+      sub_title: "Reshama Shaikh - Summary of the sprint event PyMC library team members, January and February 2022"
+      sm_image: "https://blog.dataumbrella.org/images/pymc-sprint/pymc-cover-insta.png"
+      big_image: "https://blog.dataumbrella.org/images/pymc-sprint/pymc-cover-insta.png"
```
[the remark this answers is not in the document]
Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:78]: yeah...I have made some mistakes, let me fix them

[the remark this answers is not in the document]
Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:78]: yup, it was a mistake. I have un-staged the commit, can you confirm it?

[the remark this answers is not in the document]
Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:78]: Screenshot 2023-02-14 at 9 37 59 PM

I'm getting this error here, if I force push it, the changes are not reflecting here

[the remark this answers is not in the document]
Comment (reshamas) [reply snowlake-v1.2/open-source/sprint_reports.html:78]: `git pull upstream main`

[the remark this answers is not in the document]
Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html]: sorry, I accidentally forced push the changes, which raised some merge conflicts. I have fixed it, can you give me an update.

File: snowlake-v1.2/open-source/sprint_reports.html [hunk 2]
```diff
@@ -71,6 +71,15 @@
         - scikit-learn
         - Data-Umbrella
         - Sprint-report
+
+    - title: "Reflections on the Data Umbrella PyMC February 2022 Sprint"
+      sub_title: "Reshama Shaikh - Summary of the sprint event PyMC library team members, January and February 2022"
```
Comment (reshamas) [inline snowlake-v1.2/open-source/sprint_reports.html:76]:
```suggestion
      sub_title: "Summary of the PyMC Sprints, Jan-Feb 2022"
```

File: snowlake-v1.2/open-source/sprint_reports.html [hunk 3]
```diff
@@ -71,6 +71,15 @@
...
+      sm_image: "/assets/images/data-umbrella/sprint_reports/pymc_feb_2022_reflections.png"
+      big_image: "/assets/images/data-umbrella/sprint_reports/pymc_feb_2022_reflections.png"
+      url: "https://blog.dataumbrella.org/pymc-sprint-summary"
+      filters:
+        - Data-Umbrella
+        - PyMC
```
Comment (reshamas) [inline snowlake-v1.2/open-source/sprint_reports.html:82]:
```suggestion
        - PyMC
        - Sprint-report
```

File: snowlake-v1.2/open-source/sprintlist.html [hunk 4]
```diff
@@ -73,7 +73,7 @@
       url: "https://sites.google.com/view/nyc-2020-scikit-sprint/"
       filters:
         - scikit-learn
-        - Data-Umbrella
+        - Data-Umbrella
```
Comment (reshamas) [inline snowlake-v1.2/open-source/sprintlist.html:76]:
```suggestion
        - Data-Umbrella
```

File: snowlake-v1.2/open-source/sprint_reports.html (hunk 2 above)
Comment (reshamas) [reply snowlake-v1.2/open-source/sprint_reports.html:76]: @CoffeeGeek101 
Can we move this entry so it's the second report?
(That makes it in default chronological order.)

Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html]: @reshamas I have updated the order for the entry, please check it and let me know.

File: snowlake-v1.2/open-source/sprint_reports.html [hunk 5]
```diff
@@ -32,6 +32,16 @@
...
         - Sprint-report
 
+    - title: "Reflections on the Data Umbrella PyMC February 2022 Sprint"
+      sub_title: "Summary of the PyMC Sprints, Jan-Feb 2022"
+      sm_image: "/assets/images/data-umbrella/sprint_reports/pymc_feb_2022_reflections.png"
+      big_image: "/assets/images/data-umbrella/sprint_reports/pymc_feb_2022_reflections.png"
+      url: "https://blog.dataumbrella.org/pymc-sprint-summary"
+      filters:
+        - PyMC
+        - PyMC
```
Comment (reshamas) [inline snowlake-v1.2/open-source/sprint_reports.html:42]:
```suggestion
        - Data-Umbrella
```

Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:42]: Screenshot 2023-02-14 at 11 15 35 PM

Is this okay?

Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:42]: I'll commit the changes, once you review it

Comment (reshamas) [reply snowlake-v1.2/open-source/sprint_reports.html:42]: It should have these 3 tags:
```
      filters:
        - PyMC
        - Data-Umbrella
        - Sprint-report
```

Comment (CoffeeGeek101) [reply snowlake-v1.2/open-source/sprint_reports.html:42]: Okay!!

**Comment by reshamas** [comment]
Thank you @CoffeeGeek101.  Congrats, very nice job!

@SangamSwadiK Thank you for guiding @CoffeeGeek101 with so much patience.

**Comment by CoffeeGeek101** [comment]
@SangamSwadiK @reshamas thank you for guiding me through the process until the end. **I'm grateful.**

**Pull request merged**